specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/azureWebCategory.json

```diff
@@ -0,0 +1,144 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2020-06-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/AzureWebCategories": {
+      "get": {
+        "tags": [
+          "AzureWebCategories"
+        ],
+        "operationId": "AzureWebCategories_ListAll",
+        "description": "Gets all the Azure Web Categorys in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of Azure Web Category resources.",
+            "schema": {
+              "$ref": "#/definitions/AzureWebCategoryListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all Azure Web Categorys for a given subscription": {
+            "$ref": "./examples/AzureWebCategoriesListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureWebCategoryPropertiesFormat": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the Azure Web Category resource."
+        },
+        "group": {
```

Comment:
What do these azureWebCategories represent?  Who creates them, who is going to use them, and what is that full scenario?

Reply:
Web Categories is an Azure Firewall policy feature for customers to filter http/s traffic in the form of categories for user protection. With Web Categories, the following use cases are achieved: 

 User will select  one or more azure WebCategories as a supported destination type in application rules for firewallPolicy (this linkage will be in a separate swagger PR). 

WebCategories will be a pre-created static map maintained in our RP and the list request will be serviced by NfvRP.

Reply:
Has someone in NRP approved putting this at at the top level with this resource type name?  There is no direct relationship implied between "azureWebCategories" and firewall and it could cause confusion in the future if NRP needs some other form of "webCategories" for a non-firewall scenario

Reply:
We have a similar API for GetFQDNTags. I ran the scenario with Vlad